speech/unit_tests/test_client.py

```diff
@@ -321,28 +321,40 @@ class _MockGAPICAlternative(object):
     confidence = 0.95234356
 
 
+class _MockGAPICMetadata(object):
+    type_url = None
```

Comment:
Don't mock this, just use the PB types. The excessive mocking can be problematic.

[Example][1]

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/6c5975502fcfab7cde6d9d34c433c26761505a72/speech/unit_tests/test_operation.py#L43-L73